lib/util.js

```diff
@@ -417,6 +456,59 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
 }
 
 
+function formatSet(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+  value.forEach(function(v) {
+    var str = formatValue(ctx, v,
+                          recurseTimes === null ? null : recurseTimes - 1);
```

Comment:
For readability, it's probably better to assign the result of the ternary to a variable here and below.

Aside: am I the only one who finds it moronic that `recurseTimes` is sometimes a number and sometimes not?